### PR TITLE
logo added and navbar menu item fixed

### PR DESCRIPTION
**Pull request by SijoGeorge7012**
image
image
image

### CODE REVIEW

File: assets/css/particle-theme.css [hunk 1]
```diff
@@ -3821,4 +3823,22 @@ section.pa-main-footer {
 .pa-dark-mode .menu-light-mode-panel .lm-indicator.dark-on {
 	opacity: 1;
 	left: 67px;
-}
\ No newline at end of file
+}
+
+/* MY CSS PROPERTYS*/
```
Comment (rohithkrajan) [inline assets/css/particle-theme.css]: do not put comment in ou project source code as my css property

File: dark-index.html [hunk 2]
```diff
@@ -52,112 +52,36 @@
 
 	<!-- Navigation Menu
 	================================================== -->
-	<a class="pa-mobile-main-logo" href="#"><img alt="" src="assets/images/logo-normal.png"></a>
+	<a class="pa-mobile-main-logo" href="#" id="logo1"><img alt="" src="assets/images/logo.png">BRAINCHILD TECHNOLOGY</a>
```
Comment (rohithkrajan) [inline dark-index.html]: id should not be given as logo1, 2 etc. Please choose names correctly

File: dark-index.html [hunk 3]
```diff
@@ -52,112 +52,36 @@
...
-		<a class="navbar-brand" href="#"><img alt="" src="assets/images/logo-normal.png"></a>
+		<a class="navbar-brand" href="#" id="logo1"><img alt="" src="assets/images/logo.png">BRAINCHILD TECHNOLOGY</a>
 
 		<!-- The Menu -->
 		<ul class="navbar-nav ml-auto">
 
 			<!-- Mobile Menu Logo (only use if "close" buttons are set to false in JS) -->
 			<li data-nav-custom-content class="custom-content pa-mobile-menu-logo"> 
-				<a class="pa-mobile-navbar-brand" href="#"><img alt="" src="assets/images/logo-mobile.png"></a>
+				<a class="pa-mobile-navbar-brand" href="#"><img alt="" src="assets/images/logo.png"></a>
```
Comment (rohithkrajan) [inline dark-index.html]: Please always put an alt for an image

**Review by rohithkrajan — CHANGES_REQUESTED** [review]
added comments

File: assets/css/particle-theme.css (hunk 1 above)
Comment (SijoGeorge7012) [reply assets/css/particle-theme.css]: can i change this name to LOGO & NAVBAR CSS PROPERTIES   ?

File: dark-index.html (hunk 2 above)
Comment (SijoGeorge7012) [reply dark-index.html]: Here can i change this name to brainchildlogo  ?

File: dark-index.html (hunk 3 above)
Comment (SijoGeorge7012) [reply dark-index.html]: ok

File: assets/css/particle-theme.css (hunk 1 above)
Comment (rohithkrajan) [reply assets/css/particle-theme.css]: yes

File: dark-index.html (hunk 2 above)
Comment (rohithkrajan) [reply dark-index.html]: yes something meaningful in the context

File: dark-index.html (hunk 3 above)
Comment (rohithkrajan) [reply dark-index.html]: alt is basically the text content of what is the image. so you can just put brainchild services logo

Comment (SijoGeorge7012) [reply dark-index.html]: ok

**Review by rohithkrajan — APPROVED** [review]
LGTM

**Review by wandercode — APPROVED** [review]

**Pull request merged**